<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>371</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="go">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>370</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Go</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>361</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Выборите входной файл</string>
    </property>
    <widget class="QLineEdit" name="table_name">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>40</y>
       <width>151</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Имя таблицы:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="choose_file">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Choose file</string>
     </property>
    </widget>
    <widget class="QLabel" name="path_label_in">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>341</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>361</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Выберите направление конвектирования</string>
    </property>
    <widget class="QRadioButton" name="csv_sql">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>96</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>csv-&gt;sql</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="sql_csv">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>96</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>sql-&gt;csv</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>361</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Выберите результирующий файл</string>
    </property>
    <widget class="QPushButton" name="path">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Choose file</string>
     </property>
    </widget>
    <widget class="QLabel" name="path_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>341</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="table_name_out">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>30</y>
       <width>151</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Имя таблицы:</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>757</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
